code-compass.el

```diff
@@ -567,33 +568,39 @@ code can infer it automatically."
     (when git-root
       (c/get-coupled-files-alist
        git-root
-       `(lambda (x)
+       (buffer-file-name)
+       `(lambda (x file-name)
           (message
            "Finished to update coupled files for %s and found %s coupled files."
            ,git-root
            (length x)))))))
 
+(defun c/show-coupled-files (files file-name)
+  (interactive)
```

Comment:
no need of `interactive` here because this does not do any action on the filename returned